config/cesm/machines/config_machines.xml

```diff
@@ -1313,33 +1320,39 @@
       <init_path lang="perl">/usr/share/Modules/3.2.10/init/perl.pm</init_path>
       <init_path lang="sh">/usr/share/Modules/3.2.10/init/sh</init_path>
       <init_path lang="csh">/usr/share/Modules/3.2.10/init/csh</init_path>
+      <init_path lang="python">/usr/share/Modules/3.2.10/init/python.py</init_path>
       <cmd_path lang="perl">/usr/bin/modulecmd perl</cmd_path>
+      <cmd_path lang="python">/usr/bin/modulecmd python</cmd_path>
       <cmd_path lang="csh">module</cmd_path>
       <cmd_path lang="sh">module</cmd_path>
       <modules>
 	<command name="purge"/>
-	<command name="load">perl/5.20.1</command>
-	<command name="load">comp-intel/2015.0.090</command>
-	<command name="load">mpi-sgi/mpt.2.11r13</command>
-	<command name="load">netcdf/4.1.3/intel/mpt</command>
-	<command name="load">cmake/2.8.12.1</command>
 	<command name="load">nas</command>
+	<command name="load">pkgsrc</command>
+	<command name="load">comp-intel/2016.2.181</command>
+        <command name="load">mpi-sgi/mpt.2.15r20</command>
+        <command name="load">szip/2.1.1</command>
+        <command name="load">hdf4/4.2.12</command>
+        <command name="load">hdf5/1.8.18_mpt</command>
+        <command name="load">netcdf/4.4.1.1_mpt</command>
       </modules>
     </module_system>
     <environment_variables>
       <env name="MPI_GROUP_MAX">1024</env>
       <env name="MPI_TYPE_MAX">100000</env>
+      <env name="MPI_TYPE_DEPTH">16</env>
       <env name="KMP_AFFINITY">noverbose,disabled</env>
       <env name="KMP_SCHEDULE">static,balanced</env>
-      <env name="OMP_STACKSIZE">64M</env>
+      <env name="OMP_STACKSIZE">256M</env>
     </environment_variables>
   </machine>
 
   <machine MACH="pleiades-san">
     <DESC>NASA/AMES Linux Cluster, Linux (ia64), Altix ICE, 2.6 GHz Sandy Bridge processors, 16 cores/node and 32 GB of memory, batch system is PBS</DESC>
+    <NODENAME_REGEX>pfe</NODENAME_REGEX>
```

Comment:
You shouldn't have the same NODENAME_REGEX entry for multiple machines, choose one as a default.   Or if you prefer - as of the current master the issue in check_input_data has been fixed and you no longer need any.